src/components/base/GenericTeaser/GenericTeaser.module.scss

```diff
@@ -0,0 +1,178 @@
+/*
+Förderung -> project teaser -> no padding
+*/
+
+@use "../../../styles/utilities/functions" as *;
+@use "../../../styles/mixins/mixins-typo" as *;
+@use "../../../styles/utilities/mediaqueries" as *;
+@use "../../../styles/utilities/utilities" as *;
+@use "../../../styles/mixins/mixins-shared-styles" as *;
+
+.teaser {
+  @include list-reset;
+
+  // double specifity to overwrite button style
+  &.teaser:not(.people):is(:hover) .linkText:after,
+  &.teaser:not(.people) .wrapper:is(:focus-visible) .linkText:after {
+    @include button-line-hover;
+  }
+
+  .link {
+    padding: 0;
+
+    .linkText {
+      margin-block: calc(-1 * px-to-rem(var(--spacing-m)));
+    }
+
+    svg {
+      color: var(--interaction-secondary);
+    }
+  }
+
+  .wrapper {
+    text-decoration: none;
+    color: var(--text-primary);
+    display: block;
+    height: 100%;
+
+    .title {
+      @include title-heading-4;
+
+      margin-block: 0;
+    }
+
+    .text {
+      @include body-text-m;
+
+      margin-block: 0;
+    }
+  }
+}
+
+// Institute
+.teaser.institute {
+  .logo {
+    margin-block-end: px-to-rem(var(--spacing-l));
+    border: px-to-rem(var(--border-width-s)) solid var(--border-weak);
+  }
+
+  .title {
+    margin-block: 0 px-to-rem(var(--spacing-s));
+  }
+
+  .text {
+    margin-block: 0 px-to-rem(var(--spacing-l));
+  }
+
+  .link {
+    svg {
+      color: var(--icon-primary);
+    }
+  }
+}
+
+// Project
+.teaser.project {
+  .title {
+    margin-block: 0 px-to-rem(var(--spacing-s));
+  }
+
+  .text {
+    margin-block: 0 px-to-rem(var(--spacing-xl));
+  }
+}
+
+// Magazine
+.teaser.magazine {
+  .image {
+    margin-block: 0 px-to-rem(var(--spacing-l));
+  }
+
+  .title {
+    margin-block: 0 px-to-rem(var(--spacing-s));
+  }
+}
+
+// Nework
+.teaser.network {
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .logo {
+    margin-block-end: px-to-rem(var(--spacing-xs));
+  }
+
+  .title {
+    margin-block: 0 px-to-rem(var(--spacing-xs));
+
+    @include mq($from: medium) {
+      margin-block: 0 px-to-rem(var(--spacing-s));
+    }
+
+    @include mq($from: large) {
+      margin-block: 0 px-to-rem(var(--spacing-m));
+    }
+  }
+
+  .text {
+    margin-block: 0 px-to-rem(var(--spacing-m));
+
+    @include mq($from: medium) {
+      margin-block: 0 px-to-rem(var(--spacing-l));
+    }
+
+    @include mq($from: large) {
+      margin-block: 0 px-to-rem(var(--spacing-xl));
+    }
+  }
+
+  .link {
+    margin-top: auto;
+  }
+}
+
+// People
+.teaser.people {
+  .image {
+    margin-block: 0 px-to-rem(var(--spacing-l));
+  }
+
+  .title {
+    margin-block: 0 px-to-rem(var(--spacing-s));
+  }
+
+  .text {
+    margin-block: 0 px-to-rem(var(--spacing-l));
+  }
+
+  .link {
```

Comment:
Here, we either have to set the link to display block or add wrapper element to force each link on a separate line.